src/pkg/packager/load/import_test.go

```diff
@@ -60,6 +60,10 @@ func TestResolveImports(t *testing.T) {
 			path:   "./testdata/import/flavor",
 			flavor: "pistachio",
 		},
+		{
+			name: "skeleton properties are overridden",
+			path: "./testdata/import/override",
```

Comment:
```suggestion
			name: "chart version and url properties are not overridden",
			path: "./testdata/import/chart",
```
nit: small changes to naming